### PR TITLE
Deploy heroku

### PR DESCRIPTION
**Pull request by rafaelmpessoa**
##### What this PR made?
Deploy on Heroku.

document that I relied on:
https://www.treinaweb.com.br/blog/deploy-de-uma-aplicacao-django-no-heroku/

i deployed my branch on heroku "deploy-heroku". But before this PR approved the correct branch that should be deployed is development. So this command must be executed:

git push heroku development:master

After finish the development:
git push heroku master

basically heroku create a new branch to deploy our app.
#45

### CODE REVIEW

**Comment by rafaelmpessoa** [comment]
@cesarbruschetta 

Could you check with the travis failed my pr?
i run python manage.py test -v2 and got success.

**Review by cesarbruschetta — APPROVED** [review]

**Comment by cesarbruschetta** [comment]
@rafaelmpessoa 
> Could you check with the travis failed my pr?

I entered Travis and saw the error in `sqlite3`,

```
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.7.2/lib/python3.7/site-packages/django/db/backends/utils.py", line 82, in _execute
    return self.cursor.execute(sql)
  File "/home/travis/virtualenv/python3.7.2/lib/python3.7/site-packages/django/db/backends/sqlite3/base.py", line 381, in execute
    return Database.Cursor.execute(self, query)
sqlite3.OperationalError: near "SCHEMA": syntax error
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "manage.py", line 21, in <module>
    main()
  File "manage.py", line 17, in main
    execute_from_command_line(sys.argv)
  File "/home/travis/virtualenv/python3.7.2/lib/python3.7/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/home/travis/virtualenv/python3.7.2/lib/python3.7/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/travis/virtualenv/python3.7.2/lib/python3.7/site-packages/django/core/management/commands/test.py", line 23, in run_from_argv
    super().run_from_argv(argv)
  File "/home/travis/virtualenv/python3.7.2/lib/python3.7/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/travis/virtualenv/python3.7.2/lib/python3.7/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/home/travis/virtualenv/python3.7.2/lib/python3.7/site-packages/django/core/management/commands/test.py", line 53, in handle
    failures = test_runner.run_tests(test_labels)
  File "/home/travis/virtualenv/python3.7.2/lib/python3.7/site-packages/django/test/runner.py", line 639, in run_tests
    self.teardown_databases(old_config)
  File "/home/travis/virtualenv/python3.7.2/lib/python3.7/site-packages/django_heroku/core.py", line 41, in teardown_databases
    self._wipe_tables(connection)
  File "/home/travis/virtualenv/python3.7.2/lib/python3.7/site-packages/django_heroku/core.py", line 33, in _wipe_tables
    """
  File "/home/travis/virtualenv/python3.7.2/lib/python3.7/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/home/travis/virtualenv/python3.7.2/lib/python3.7/site-packages/django/db/backends/utils.py", line 76, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/travis/virtualenv/python3.7.2/lib/python3.7/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/home/travis/virtualenv/python3.7.2/lib/python3.7/site-packages/django/db/utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/travis/virtualenv/python3.7.2/lib/python3.7/site-packages/django/db/backends/utils.py", line 82, in _execute
  File "/home/travis/virtualenv/python3.7.2/lib/python3.7/site-packages/django/db/backends/sqlite3/base.py", line 381, in execute
    return Database.Cursor.execute(self, query)
django.db.utils.OperationalError: near "SCHEMA": syntax error
```
https://travis-ci.org/cesarbruschetta/api-fintech/builds/533832278

**Comment by cesarbruschetta** [comment]
@rafaelmpessoa 

the problem was tired by the configuration of heroku, I changed the configuration below and the complete description of the problem is in the link that I added in the issue below

```
# Configure Django App for Heroku.
django_heroku.settings(locals(), test_runner=False)
```

### Ref
https://stackoverflow.com/a/51209478/10342534

**Review by julianeventeu — APPROVED** [review]
